### DISCUSSION TITLE
Des petites indications (more tooltip)

### DISCUSSION THREAD

**Discussion by Klelian** [skarab42/marv#26] [Ideas]
Pour mieux comprendre à quoi correspondent les items, ce serait cool d'implémenter des indications explicatives lorsqu'on passe la souris dessus. (toutes les fonctions n'ont pas besoin de grosses explications dans un doc annexe, parfois un petit tips suffit :) )

merci pour le super travail !

**Comment by syrm** [comment]
Tu as des exemples à donner afin que ça soit plus parlant ?

**Comment by Klelian** [comment]
par exemple quand on ajoute des animations, on peut choisir dans quel évènement, mais "part" ou "join" c'est pas parlant c'est peut etre juste que la traduction est pas tout à fait terminée, mais ce genre de choses.

**Reply by skarab42** [reply]
Dans un premier temps je vais lister ça dans la documentation. Mais a terme j'ajouterais plus de tooltip ;)
